packages/contracts/contracts/ProxyAdmin.sol

```diff
@@ -35,7 +36,7 @@ contract ProxyAdmin {
      * @param _registry     Address of the ChugSplashRegistry.
      * @param _proxyUpdater Address of the ProxyUpdater.
      */
-    constructor(ChugSplashRegistry _registry, address _proxyUpdater) {
+    constructor(ChugSplashRegistry _registry, address _proxyUpdater) Owned(msg.sender) {
```

Comment:
Owner of this is the `ChugSplashManager`, so I think we need to put a function into the `ChugSplashManager` that calls `transferProxyOwnership`. I'm also thinking now that it might be better to merge the `ChugSplashManager` and `ProxyAdmin` contracts because this ownership stuff starts to get complex (and for good reason). It creates a lot of extra functions in both the `ProxyAdmin` and the `ChugSplashManager`. I'm also coming around to the idea that a malicious `ProxyAdapter` would be able to completely mess with the proxies anyway, so it's not like this separation really gives you any extra security.

Reply:
Agreed, made #53 to merge the `ProxyAdmin` into the `ChugSplashManager`. Will close this PR and open a new one that can be merged after #53 is approved.

Reply:
Approved #53!

Reply:
Sweet! Closed by #55.